src/catalog/catalog-urls.md

```diff
@@ -79,3 +79,23 @@ To prevent search engines from indexing multiple URLs that lead to the same cont
 
    ![]({% link images/images/msg-cache-management.png %}){: .zoom}
    [_Refresh Cache_]({% link system/cache-management.md %})
+
+## Configure catalog media URL format
+
+1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
+
+1. In the left panel, expand **General** and choose **Web** underneath.
+
+1. Expand ![]({% link images/images/btn-expand.png %}) the **Url Options** section and set the options:
+
+![Web > General Options]({% link images/images/config-general-web-url-options.png %}){: .zoom}
+[_URL Options configuration settings_]({% link stores/store-urls.md %})
+
+{: .field-table }
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|Add Store Code to URLs|Global|If Web Server Rewrites are enabled, inserts the Store Code of the current view in the URL. Options: Yes / No|
+|Auto-redirect to Base URL|Global|(For single-store setups) If there is a broken link on your site, redirects traffic to the base URL, rather than to a page with a “404 Page Not Found” message. Options: No / Yes (302 Found) / Yes (301 Moved Permanently) <br /><br />**_Important!_** Do not use auto-redirect to base URL for multi-store setups.|
+|Catalog media URL format|Defines the [URLs format]({% link catalog/catalog-urls.md %}) assigned to products and categories. Options:
+**Unique hash per image variant (Legacy mode)** - Determine converting filename to a unique hash value.
```

Comment:
I don't understand this phrase "Determines converting filename to a unique hash value." Is that supposed to mean something like "Defines converted filename as a unique hash value."?

Reply:
Yes, @shrielenee, you totally right :) I'll implement your more clear variant with the next commit. Thank you!

Reply:
Done, please check! ☺️